backbone/src/main/AndroidManifest.xml

```diff
@@ -4,5 +4,7 @@
 
     <!-- We allow users to secure their data using their fingerprint see -->
     <uses-permission android:name="android.permission.USE_FINGERPRINT"/>
+    <!-- ActiveSteps may vibrate the phone on start and on finish -->
+    <uses-permission android:name="android.permission.VIBRATE"/>
```

Comment:
That's a good point.  I'll comment it out and write that it is the responsibility of the app to include.